File: Java/src/pl/eco/roslina/Jagody.java
package pl.eco.roslina;

import pl.eco.Organizm;
import pl.eco.Roslina;
import pl.eco.Swiat;

public class Jagody extends Roslina {
    public Jagody(int x, int y, Swiat moj) {
        sila = 99;
        inicjatywa = 0;
        nazwa = "wilcze jagody";
        symbol = '*';
        wiek = 0;
        this.x = x;
        this.y = y;
        this.moj = moj;
    }
    public Jagody(int x, int y,Swiat moj,int wiek){
        sila = 99;
        inicjatywa = 0;
        nazwa = "wilcze jagody";
        symbol = '*';

        this.wiek = wiek;
        this.x = x;
        this.y = y;
        this.moj = moj;
    }

    @Override
    public boolean obrona(Organizm atujacy) {
        this.czyZyje=false;
        return false;
    }
}

File: Python/Rosliny/Jagody.py
from Roslina import Roslina


class Jagody(Roslina):
    def __init__(self, x, y, moj):
        self.sila = 99
        self.inicjatywa = 0
        self.symbol = '*'
        self.nazwa = "wilcze jagody"
        self.wiek = 0
        self.x = x
        self.y = y
        self.moj = moj
        self.czyZyje = True

    def generator(self, x, y):
        return Jagody(x, y, self.moj)

    def obrona(self, atakujacy):
        self.czyZyje = False
        return False
